code/modules/vore/eating/living_vr.dm

```diff
@@ -587,3 +587,87 @@
 	var/new_color = input(src,"Select a new color","Body Glow",glow_color) as color
 	if(new_color)
 		glow_color = new_color
+
+/mob/living/proc/eat_trash()
+	set name = "Eat Trash"
+	set category = "Abilities"
+	set desc = "Consume held garbage."
+
+	var/obj/item/I = get_active_hand()
+	if(!I)
+		to_chat(src, "<span class='notice'>You are not holding anything.</span>")
+		return
+	var/list/edible_trash = list(/obj/item/trash,
+				/obj/item/device/flashlight,
+				/obj/item/toy/figure,
+				/obj/item/weapon/cigbutt,
+				/obj/item/weapon/bananapeel,
+				/obj/item/stack/material/cardboard,
+				/obj/item/weapon/light,
+				/obj/item/weapon/paper,
+				/obj/item/weapon/broken_bottle,
+				/obj/item/weapon/reagent_containers/food,
+				/obj/item/inflatable/torn,
+				/obj/item/weapon/flame/match,
+				/obj/item/clothing/mask/smokable,
+				/obj/item/weapon/storage/wallet,
+				/obj/item/weapon/spacecash,
+				/obj/item/broken_device,
+				/obj/item/weapon/corncob,
+				/obj/item/weapon/paperplane,
+				/obj/item/weapon/pen,
+				/obj/item/weapon/photo,
+				/obj/item/weapon/storage/box/wings,
+				/obj/item/weapon/storage/box/matches,
+				/obj/item/weapon/storage/fancy/cigarettes,
+				/obj/item/weapon/storage/fancy/egg_box,
+				/obj/item/weapon/storage/fancy/candle_box,
+				/obj/item/weapon/storage/fancy/crayons,
+				/obj/item/weapon/dice,
+				/obj/item/weapon/newspaper,
+				/obj/item/weapon/reagent_containers/glass/rag)
+	for(var/type in edible_trash)
+		if(istype(I,type))
```

Comment:
Can drop the for() and do `is_type_in_list(I,edible_trash)` (or the global suggestion)